src/logic/map_objects/tribes/tribe_descr.cc

```diff
@@ -84,6 +87,15 @@ TribeDescr::TribeDescr(const LuaTable& table,
 		load_roads("busy", &busy_road_paths_);
 		load_roads("waterway", &waterway_paths_);
 
+#define LOAD_BRIDGE_IF_PRESENT(dir, type)                                                          \
```

Comment:
Please use a lambda function instead if you can; they are easier to read.

Reply:
Ok